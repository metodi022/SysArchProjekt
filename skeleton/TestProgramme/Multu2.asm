lui $1,0x1010
lui $2,0x1010
ori $1,0x0000
addiu $2, $2, 0x0001
multu $1, $2
mfhi $30
mflo $31