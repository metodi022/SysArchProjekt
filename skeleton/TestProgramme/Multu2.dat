3c011010
3c021010
34210000
24420001
00220019
0000f010
0000f812
