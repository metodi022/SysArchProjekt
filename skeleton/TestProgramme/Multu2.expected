@1 10100000
@2 10100001
@1f 01020100
@1e 10100000
